cmd/infracost/run.go

```diff
@@ -564,11 +564,11 @@ func (r *parallelRunner) fetchProjectUsage(projects []*schema.Project) map[*sche
 	}
 
 	spinnerOpts := ui.SpinnerOptions{
-		EnableLogging: r.runCtx.Config.IsLogging(),
+		EnableLogging: false,
```

Comment:
I don't think this does what you want it to do. It is used to switch between the spinner and logging, so if it's set to false then the spinner still runs and we end up with output like this:
```
2024-03-26T22:33:24Z INF Starting: Evaluating Terraform directory
  ✔ Retrieving usage defaults for 9 resources from Infracost Cloud
2024-03-26T22:33:25Z INF Starting: Retrieving cloud prices to calculate costs
```

I think we should keep these ones as they were anyway since they are useful log lines and they don't add much noise. The main noise has been removed by removing the 'Hiding resource with no usage' lines.